### PR TITLE
Add a sentence to contact section

### PR DESCRIPTION
**Pull request by theotsuaneng**
Add a sentence to contact section

### CODE REVIEW

**Comment by theotsuaneng** [comment]
Screenshot (59)

File: dark/assets/css/styles.css [hunk 1]
```diff
@@ -913,7 +913,7 @@ html {
     display: block;
     width: 130px;
     height: 2px;
-    background: rgb(168, 168, 168);
+    background: rgb(209, 38, 38);
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:916]: Remove this color bcs the current color is looks good @theotsuaneng

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 2]
```diff
@@ -232,8 +232,12 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
             <h1>CONTACT ME</h1>
             <h3>CONTACT <span class="text-info">ME</span></h3>
             <p class="contact-text">
-                    📚 Education : St. Anthony's College, Kandy
+                    📚 Education : St. Anthony's College, Kandy.
```
Comment (YoshithaRathnayake) [inline dark/index.html:235]: Remove this full stop from this @theotsuaneng 

```suggestion
                    📚 Education : St. Anthony's College, Kandy

```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -90,7 +90,7 @@ <h1><span class="auto-type"></span></h1>
         </div>
 
     </section>
-
+    <p>Contact me via Email, Message or connect me with social media accounts. Also send me any inquiries by filling in the form given just below</p>
```
Comment (YoshithaRathnayake) [inline index.html:93]: Remove the sentence from the index.html bcs index.html means the pre-loader of this website @theotsuaneng

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by theotsuaneng** [comment]
Hi @YoshithaRathnayake I have resolved the stated issues.

File: dark/index.html [hunk 4]
```diff
@@ -234,6 +234,10 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
             <p class="contact-text">
                     📚 Education : St. Anthony's College, Kandy
                 </p>
+                <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+                <i class="fa fa-envelope" style="font-size:24px"></i>
+                <p>Contact me via Email, Message or connect me with social media accounts.
```
Comment (YoshithaRathnayake) [inline dark/index.html:239]: Can you align center the text @theotsuaneng

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 5]
```diff
@@ -234,8 +234,13 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
             <p class="contact-text">
                     📚 Education : St. Anthony's College, Kandy
                 </p>
+                <center>
                 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
                 <i class="fa fa-envelope" style="font-size:24px"></i>
+                </center>
+                <style>
```
Comment (YoshithaRathnayake) [inline dark/index.html:241]: Don't use inline css just add wrap the text with a `center` tag

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by theotsuaneng** [comment]
@YoshithaRathnayake the css has been removed may you accept the pull.

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @theotsuaneng

**Pull request merged**